zetaclient/orchestrator/bootstap_test.go

```diff
@@ -0,0 +1,492 @@
+package orchestrator
```

Comment:
nit - Can we consider maybe having a separate package for the test? `orchestrator_test`

Reply:
Go distinguishes $package and $package_test. Generally, I'd use $package_test if regular $package conflicts with imports.